samples/matlab_new/reactor1.m

```diff
@@ -0,0 +1,85 @@
+function reactor1(g)
```

Comment:
Broken
```
>> reactor1
  reactor1 Zero-dimensional kinetics: adiabatc, constant pressure.
 
  This example illustrates how to use class 'Reactor' for zero-dimensional
  kinetics simulations. Here the parameters are set so that the reactor is
  adiabatic and very close to constant pressure.

Error using calllib
Method was not found.

Error in Reactor (line 55)
            r.id = calllib(ct, 'reactor_new2', typ);

Error in IdealGasReactor (line 12)
    r = Reactor(contents, 'IdealGasReactor');

Error in reactor1 (line 27)
    r = IdealGasReactor(gas);
```